### PR TITLE
Implemented AWS tag filters

### PR DESCRIPTION
**Pull request by akhil-sumologic**

### CODE REVIEW

File: sumologic-app-utils/src/sumoresource.py [hunk 1]
```diff
@@ -336,6 +365,10 @@ def _get_path(self, props):
                 path["limitToRegions"] = regions
             if "Namespaces" in props:
                 path["limitToNamespaces"] = props.get("Namespaces")
+                print("limitToNamespaces: ", path["limitToNamespaces"])
```
Comment (himsharma01) [inline sumologic-app-utils/src/sumoresource.py:368]: Print can be removed

File: sumologic-app-utils/src/sumoresource.py [hunk 2]
```diff
@@ -317,6 +319,33 @@ def build_source_params(self, props, source_json=None):
...
+            except json.JSONDecodeError as e:
+                print("Invalid AWS tag filters JSON:", e)
+                aws_tag_filters = {}
+            except jsonschema.exceptions.ValidationError as e:
+                print(f"JSON validation error: {e.message}")
+                aws_tag_filters = {}
+        else:
+            aws_tag_filters = {}
+        for key, value in aws_tag_filters.items():
+            if key in namespaces:
```
Comment (himsharma01) [inline sumologic-app-utils/src/sumoresource.py:341]: Lets only filter aws namespaces

**Review by himsharma01 — CHANGES_REQUESTED** [review]
lgtm

File: sumologic-app-utils/src/sumoresource.py (hunk 2 above)
Comment (akhil-sumologic) [reply sumologic-app-utils/src/sumoresource.py]: Addressed

**Review by himsharma01 — APPROVED** [review]
lgtm

**Review by himsharma01 — APPROVED** [review]
lgtm

**Review by himsharma01 — APPROVED** [review]
lgtm

**Pull request merged**